### ISSUE TITLE
Source level labels as proxy for news label

### ISSUE THREAD

**Issue by SaffronWolf** [MELALab/nela-gt#3]
First off, thank you so much for your efforts in making available NELA GT datasets.. I would like to extend the use case of the dataset to news classification. Does it make sense use source-level labels directly as news-level label? I understand that labelling wont be 100% accurate but would it be accurate enough to be used for fake news classification problems?

**Comment by mgruppi** [comment]
Hello! 
I assume what you are talking about is to use the source labels as article-level labels. This has been done before (see https://ojs.aaai.org/index.php/ICWSM/article/view/14976 and https://ojs.aaai.org/index.php/ICWSM/article/view/14976 and more recently https://arxiv.org/abs/2101.10973).

However, I would argue that doing so would create many noisy samples because unreliable sources will not always publish articles that are false. They will even copy entire articles from reliable sources and republish them.

I suggest that, if you do that, make sure to split articles and sources into in-training and out-of-training sets to test for robustness.

**Comment by SaffronWolf** [comment]
Thanks for the reply. 

> I suggest that, if you do that, make sure to split articles and sources into in-training and out-of-training sets to test for robustness

As in split into train and test sets?

**Comment by mgruppi** [comment]
> Thanks for the reply.
> 
> > I suggest that, if you do that, make sure to split articles and sources into in-training and out-of-training sets to test for robustness
> 
> As in split into train and test sets?

Similar to a train-test split, but in addition to choosing some articles for training and some for testing, you leave all the articles of a specific source S out of the training phase. This is important to ensure your classifier can predict articles from sources it has never seen, or if it is just learning to separate the sources it knows.
